### DISCUSSION TITLE
Custom DKIM Prefix?

### DISCUSSION THREAD

**Discussion by james-william-r** [postalserver/postal#2903] [Help with using Postal] [answered]
Just wondering if any settings make it possible for us to alter the `postal-` prefix on DKIM hosts? ☺️

**Comment by willpower232** [comment]
Not through the UI but presumably you could tinker with the database records and see how you get on. As long as the key is still the same and correct, I don't think there would be an issue.

**Comment by adamcooke** [comment, marked as answer]
Yes: https://github.com/postalserver/postal/blob/main/doc/config/yaml.yml#L144-L145

**Reply by james-william-r** [reply]
Sweeet, thanks @adamcooke! 🤘
